theme.ts

```diff
@@ -0,0 +1,21 @@
+// Ignore this rule because tailwind is all cjs and we need to use requires here
+// Maybe come back to this but this seems to be the easy way out
+/* ts-ignore:@typescript-eslint/no-var-requires */
```

Comment:
The other option is just to use JavaScript here. Alternatively, I'm not against just keeping this logic in `tailwind.config.cjs` for now. Other than the `omit` function that we're not actively using right now, we're not doing anything in this file.

Reply:
Particularly because this is the file that is [failing the build](https://github.com/temporalio/ui/runs/4361170444?check_suite_focus=true#step:4:17).

Reply:
I was waiting on choosing the colors later with Candace and brought the omit function because I figured we'd end up using it. I'll probably pick instead of omit in our more final version.